### PR TITLE
[monitoring/prober] Move prober Dockerfile into the common monitoring image

### PR DESCRIPTION
**Pull request by pratibhagupta2109**

### CODE REVIEW

File: test/docker_e2e.sh [hunk 1]
```diff
@@ -62,32 +62,32 @@ docker rm -f dss-crdb-for-debugging &> /dev/null || echo "No CRDB to clean up"
 
 echo "Starting cockroachdb with admin port on :8080"
 docker run -d --rm --name dss-crdb-for-debugging \
-	-p 26257:26257 \
-	-p 8080:8080 \
-	cockroachdb/cockroach:v21.2.7 start-single-node \
-	--insecure > /dev/null
+-p 26257:26257 \
```
Comment (BenjaminPelletier) [inline test/docker_e2e.sh:65]: Let's not remove the indents on these multi-line commands

**Review by BenjaminPelletier — COMMENTED** [review]
I think this PR looks good after the spacing adjustment (see other comment), but this branch is currently out of sync with master -- it needs the master branch to be merged into it.

File: test/docker_e2e.sh (hunk 1 above)
Comment (pratibhagupta2109) [reply test/docker_e2e.sh]: Got tempered by my ide. Fixed it now.

**Comment by BenjaminPelletier** [comment]
I think this looks good, but this branch is currently out of sync with master -- it needs the master branch to be merged into it (see the note at the bottom of this page where it says "This branch has conflicts that must be resolved")

**Comment by pratibhagupta2109** [comment]
> 

Fixed. Thank you.

**Review by BenjaminPelletier — APPROVED** [review]

**Pull request merged**